src/main/kotlin/com/jetbrains/snakecharm/lang/parser/SmkStatementParsing.kt

```diff
@@ -205,13 +206,22 @@ class SmkStatementParsing(
             parseRuleParameter(section)
         } else {
             nextToken()
-            incompleteRule = !checkMatches(PyTokenTypes.INDENT, "Indent expected...")
+            //incompleteRule = !checkMatches(PyTokenTypes.INDENT, "Indent expected...")
+            incompleteRule = !atToken(PyTokenTypes.INDENT)
             if (!incompleteRule) {
+                nextToken()
                 while (!atToken(PyTokenTypes.DEDENT)) {
                     if (!parseRuleParameter(section)) {
                         break
                     }
                 }
+            } else {
+                if (section.parameterListStatement != SmkElementTypes.RULE_OR_CHECKPOINT_ARGS_SECTION_STATEMENT) {
+                    // Only rules, checkpoints and use rules can have an empty statement list
+                    myBuilder.error(PyPsiBundle.message("indent.expected"))
```

Comment:
Is `USE_ARGS_SECTION_STATEMENT` missing here?

According to impl this method should be called for `Use` blocks:
```kotlin
tt in SmkTokenTypes.RULE_LIKE -> parseRuleLikeDeclaration(getSectionParsingData(tt!!))
```
and use block expects `SmkElementTypes.USE_ARGS_SECTION_STATEMENT` token type:

```kotlin
   private val useSectionParsingData = SectionParsingData(
        declaration = USE_DECLARATION_STATEMENT,
        name = "use",
        parameterListStatement = SmkElementTypes.USE_ARGS_SECTION_STATEMENT,
        sectionKeyword = SmkTokenTypes.USE_KEYWORD
    )
```